ui/server/src/main/scala/pl/touk/nussknacker/ui/config/FeatureTogglesConfig.scala

```diff
@@ -42,6 +43,7 @@ object FeatureTogglesConfig extends LazyLogging{
     val tabs = parseOptionalConfig[List[TopTab]](config, "tabs")
     val intervalTimeSettings = config.as[IntervalTimeSettings]("intervalTimeSettings")
     val testDataSettings = config.as[TestDataSettings]("testDataSettings")
+    val skipArchiveRedirect = config.getAs[Boolean]("skipArchiveRedirect")
```

Comment:
can we make it backward compatible so that if no skipArchiveRedirect in configuration everything works fine ?

Reply:
I think it is right now - `getAs[A]` works like `as[Option[A]]` so if config path is not defined we will end up with `None`

Reply:
👍